packages/cinterop/src/darwin/kotlin/io/realm/internal/interop/RealmInterop.kt

```diff
@@ -1065,7 +1065,6 @@ actual object RealmInterop {
     }
 
     private fun createSingleThreadDispatcherScheduler(dispatcher: CoroutineDispatcher): CPointer<realm_scheduler_t>? {
-        printlntid("createSingleThreadDispatcherScheduler")
```

Comment:
These `printlntid` is only for debugging purpose and are no-ops unless you override `printlntid`. See definition of `printlntid` for the details.